lib/mongoid/criteria/queryable/selectable.rb

```diff
@@ -69,8 +69,9 @@ def all(*criteria)
         # @example Add the criterion.
         #   selectable.and({ field: value }, { other: value })
         #
-        # @param [ Array<Hash | Criteria> ] criteria Multiple key/value pair
-        #   matches or Criteria objects that all must match to return results.
+        # @param [ [ Hash | Criteria ]... ] *criteria
```

Comment:
This should be `Hash | Criteria | Array<Hash | Criteria>`

Reply:
Noted